protos/node/node.proto

```diff
@@ -35,37 +35,38 @@ message Node {
     float longitude = 4;
   }
   string table_name = 1 [(gogoproto.moretags) = "pg:\"node_metrics,alias:t,discard_unknown_columns\""];
-  string node_id = 2;
-  string btfs_version = 3;
-  uint64 up_time = 4;
-  uint64 storage_used = 5;
-  uint64 storage_volume_cap = 6;
-  uint64 memory_used = 7;
-  double cpu_used = 8;
-  uint64 upload = 9;
-  uint64 download = 10;
-  uint64 total_upload = 11;
-  uint64 total_download = 12;
-  uint64 storage_price_deal = 13;
-  uint64 bandwidth_price_deal = 14;
-  Settings settings = 15 [
+  uint64 id = 2;
```

Comment:
append at the end, do not change position of everything

Reply:
i c.

Reply:
maybe we should also add a `json:"-"` to this field since it should *never* be exposed to public, thoughts @wxue ?

Reply:
I mean, a specialized field that is not defined as a valid protobuf field, but only generated for pg tags